deploy/role.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
 rules:
 - apiGroups:
   - "extensions"
```

Comment:
which part of the code still requires access to `extensions` apiGroup, for `ingresses` resources?

Reply:
Yes extensions grp can be removed as we can still manipulated existing resource using ingress.networking.v1 call.